code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -167,9 +167,10 @@
 	name = "jump boots"
 	desc = "A specialized pair of combat boots with a built-in propulsion system for rapid foward movement."
 	icon_state = "jetboots"
-	item_state = "jackboots"
+	item_state = "jetboots"
 	item_color = "hosred"
 	resistance_flags = FIRE_PROOF
+	pockets = /obj/item/weapon/storage/internal/pocket/shoes
```

Comment:
im not sure that this is how you do this, you want the pocket to be quickdraw. this might already do that im not sure